README.md

```diff
@@ -1,236 +1,139 @@
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
-- [📝 License](#license)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[bookstore\] ](#-bookstore-)
+  - [Learning objectives ](#learning-objectives-)
+  - [🛠 Built With ](#-built-with-)
+    - [Key Features ](#key-features-)
+    - [Home page](#home-page)
+  - [🚀 Live Demo and video ](#-live-demo-and-video-)
+  - [💻 Getting Started ](#-getting-started-)
+  - [Test](#test)
+    - [Prerequisites](#prerequisites)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+    - [Key Features ](#key-features--1)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [bookstore] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+**[bookstore]** This is a simple book store app built with React js. It uses the Google Books API to search for books and display them. It also uses react-router to navigate between pages.
+## Learning objectives <a name="objectives"></a>
 
-**[your_project__name]** is a...
+- Use React Router.
+- Use React components.
 
 ## 🛠 Built With <a name="built-with"></a>
-
-### Tech Stack <a name="tech-stack"></a>
-
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
-  </ul>
-</details>
-
-<details>
-  <summary>Server</summary>
-  <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
-  </ul>
-</details>
-
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
+- React
+- JavaScript
+- React
+- JSX
+- CSS
+  
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
+-[ ] Vite React App
+-[ ] reusable components: /components.
+-[ ] Setup your router inside of <App>
+-[ ] Add 2 routes in your application, each should render a page component:
+- /
+- /categories
+-[ ] Contains individual book state, as well as a button to delete the book
+-[ ] Contains a list to renders individual books
+-[ ] Contains a form to create new books
+-[ ] Contains a navigation to link to the 2 routes you created
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+<!-- LIVE DEMO AND THE DEPLOY -->
+### Home page
 
-## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+## 🚀 Live Demo and video <a name="live-demo"></a>
+https://jkanyi-web.github.io/bookstore-app/dist
+https://jkanyi-web.github.io/bookstore-app/
 
-- [Live Demo Link](https://google.com)
+- [Click Here]()
```

Comment:
- [ ] Since the project has not been deployed yet, I suggest that you remove these links under the **Live Demo and video** section to make your README file look more professional.